Tweetinvi.Core/Public/Models/Enum/Language.cs

```diff
@@ -3,233 +3,198 @@
     /// </summary>
     public enum Language
     {
-        [Language("undefined", "xx")] Undefined = 0,
-        [Language("poi")] Poi,
-        [Language("ab")] Abkhaz,
-        [Language("aa")] Afar,
-        [Language("af")] Afrikaans,
-        [Language("ak")] Akan,
-        [Language("sq")] Albanian,
-        [Language("am")] Amharic,
-        [Language("ar")] Arabic,
-        [Language("an")] Aragonese,
-        [Language("hy")] Armenian,
-        [Language("as")] Assamese,
-        [Language("av")] Avaric,
-        [Language("ae")] Avestan,
-        [Language("ay")] Aymara,
-        [Language("az")] Azerbaijani,
-        [Language("bm")] Bambara,
-        [Language("ba")] Bashkir,
-        [Language("eu")] Basque,
-        [Language("be")] Belarusian,
-        [Language("bn")] Bengali,
-        [Language("bn")] Bangla,
-        [Language("bh")] Bihari,
-        [Language("bi")] Bislama,
-        [Language("bs")] Bosnian,
-        [Language("br")] Breton,
-        [Language("bg")] Bulgarian,
-        [Language("my")] Burmese,
-        [Language("ca")] Catalan,
-        [Language("ca")] Valencian,
-        [Language("ch")] Chamorro,
-        [Language("ce")] Chechen,
-        [Language("ny")] Chichewa,
-        [Language("ny")] Chewa,
-        [Language("ny")] Nyanja,
-        [Language("zh")] Chinese,
-        [Language("cv")] Chuvash,
-        [Language("kw")] Cornish,
-        [Language("co")] Corsican,
-        [Language("cr")] Cree,
-        [Language("hr")] Croatian,
-        [Language("cs")] Czech,
-        [Language("da")] Danish,
-        [Language("dv")] Divehi,
-        [Language("dv")] Dhivehi,
-        [Language("dv")] Maldivian,
-        [Language("nl")] Dutch,
-        [Language("dz")] Dzongkha,
-        [Language("en")] English,
-        [Language("eo")] Esperanto,
-        [Language("et")] Estonian,
-        [Language("ee")] Ewe,
-        [Language("fo")] Faroese,
-        [Language("fj")] Fijian,
-        [Language("fi")] Finnish,
-        [Language("fr")] French,
-        [Language("ff")] Fula,
-        [Language("ff")] Fulah,
-        [Language("ff")] Pulaar,
-        [Language("ff")] Pular,
-        [Language("gl")] Galician,
-        [Language("ka")] Georgian,
-        [Language("de")] German,
-        [Language("el")] Greek,
-        [Language("el")] Modern,
-        [Language("gn")] Guaraní,
-        [Language("gu")] Gujarati,
-        [Language("ht")] Haitian,
-        [Language("ht")] HaitianCreole,
-        [Language("ha")] Hausa,
-        [Language("he")] Hebrew,
-        [Language("hz")] Herero,
-        [Language("hi")] Hindi,
-        [Language("ho")] HiriMotu,
-        [Language("hu")] Hungarian,
-        [Language("ia")] Interlingua,
-        [Language("id")] Indonesian,
-        [Language("ie")] Interlingue,
-        [Language("iw")] Ivrit,
-        [Language("ga")] Irish,
-        [Language("ig")] Igbo,
-        [Language("ik")] Inupiaq,
```

Comment:
Wherever a language has multiple common names, I tried to select the most commonly used one (not an exact science) but generally went with whatever the title of the Wikipedia article about the language was.
In this case that's Inupiat (https://en.wikipedia.org/wiki/Inupiat_language).

My use case requires being able to display a list of unique languages to my end-user, listed with their names in English. The name of the language that should be displayed should be whatever they're most likely to recognise, hence whatever it's commonly known as. So this is an area where your current solution wouldn't meet my requirements as there are multiple entries with the language code "ik", so no obvious way to select the most common name of the language.

Perhaps a compromise would be to list all the language names (using the array as you've done), with the most common first (to meet my use case), and prevent duplicate usage of the same language code (no need if each enum entry can have multiple language names and also as it removes the ambiguity of which enum entry should be returned as the language of a tweet).